config/services.php

```diff
@@ -23,6 +23,14 @@
         'secret' => env('MANDRILL_SECRET'),
     ],
 
+    'northstar' => [
+        'grant' => 'password',
+        'url' => env('NORTHSTAR_URL'),
+        'client_id' => env('NORTHSTAR_CLIENT_ID'),
+        'client_secret' => ent('NORTHSTAR_CLIENT_SECRET'),
+        'scope' => ['user', 'admin'],
```

Comment:
Ah got it! And is this because it is assigning roles to the user instead of having a user and an admin?